README.md

```diff
@@ -10,32 +10,29 @@
 
 > This is currently undergoing significant changes as the remaster is being built from the legacy app. More updates will be coming soon!
 
-To help contribute, currently go to the [Legacy App](https://github.com/wanderers-guide/wanderers-guide) and follow the README there. If you would like to contribute to the rework, come chat to us on the [Wanderers Guide Discord](https://discord.gg/kxCpa6G) and search tag `@developer` in the development channel.
+To help contribute, go to the [Legacy App](https://github.com/wanderers-guide/wanderers-guide-legacy/tree/main/services/express) and follow the README there. If you would like to contribute to the rework, come chat to us on the [Wanderers Guide Discord](https://discord.gg/kxCpa6G) and search tag `@developer` in the development channel.
 
 ## Setup
 
 1. Install node.js using the instructions here: <https://nodejs.org/en/download>
 2. Install the supabase CLI for your OS following the instructions: <https://supabase.com/docs/guides/cli/getting-started>
-3. Run `supabase start` to initialize supabase locally
-4. Go into the `/data` folder and run `./create-db.sh postgresql://postgres:postgres@127.0.0.1:54322/postgres`. This will create all the tables needed
-5. Go into the `/frontend` directory
-6. Copy `.env.local.template` to `.env.local` and paste the supabase url and anon key into `.env.local`. The default url to the supabase panel is <http://127.0.0.1:54323/>
+3. Run `supabase start` to initialize supabase locally, this will print a few details about the supabase local development setup that will be required later
+4. Go into the `data/` folder and run `./create-db.sh <DB_URL>` (replace `<DB_URL>` with value obtained in step 3). This will create all the tables needed
```

Comment:
Unless someone changes the Postgres port I'd recommend adding the default value here, like you did with the default studio URL

Reply:
Good point, I wasn't sure how often this would change, but assuming it doesn't I'm happy to just add the default value in here

Reply:
The default port is configured in the `supabase/config.toml` file, that can be something we look out for if changes to update the docs.